### PR TITLE
Add section about the Liquibase upgrade in the release notes

### PR DESCRIPTION
**Pull request by sguilhen**

### CODE REVIEW

File: upgrading/topics/keycloak/changes-17_0_0.adoc [hunk 1]
```diff
@@ -22,6 +22,18 @@ A number of things have changed since the preview Quarkus distribution was relea
...
+
 = Client Policies Migration : client-scopes
 
 If you used a policy including client-scopes condition and edited JSON document directly, you will need to change the "scope" field name in a JSON document to "scopes".
+
+= Liquibase upgraded to version 4.6.2
+
+Liquibase was updated from version 3.5.5 to 4.6.2, which includes, among other things, several bug fixes, a new way of registering custom extensions
+via ServiceLoader, and CockroachDB support. The latter is specially important for the new storage's JPA implementation as we aim
+to have CockroachDB as one of the supported databases for the new storage.
```
Comment (hmlnarik) [inline upgrading/topics/keycloak/changes-17_0_0.adoc]:
```suggestion
Liquibase was updated from version 3.5.5 to 4.6.2, which includes, among other things, several bug fixes, and a new way of registering custom extensions
via `ServiceLoader`.
```

File: upgrading/topics/keycloak/changes-17_0_0.adoc [hunk 2]
```diff
@@ -22,6 +22,18 @@ A number of things have changed since the preview Quarkus distribution was relea
...
+
+Migration from previous Keycloak versions to Keycloak 17 has been extensively tested with all the currently supported databases,
+but we would like to stress that *backing up existing databases is really important before attempting to migrate*. While we did our best
+to test all the implications of the Liquibase upgrade, it required a fairly big change throughout the code and it is possible
+that users run into unexpected issues when upgrading.
```
Comment (hmlnarik) [inline upgrading/topics/keycloak/changes-17_0_0.adoc]: Suggestion:
```suggestion
but we would like to stress the importance of closely following Upgrading guide, specifically of *backing up existing database before upgrade*. While we did our best
to test the consequences of the Liquibase upgrade, some installations could be using specific setup unknown to us.
```

**Comment by sguilhen** [comment]
@andymunro Can you please review these changes?

File: upgrading/topics/keycloak/changes-17_0_0.adoc [hunk 3]
```diff
@@ -22,6 +22,16 @@ A number of things have changed since the preview Quarkus distribution was relea
...
 * Logging level and format is now configured with `--log-level` and `--log-format`, while in the past these had to be configured using unsupported Quarkus properties.
+
 = Client Policies Migration : client-scopes
 
 If you used a policy including client-scopes condition and edited JSON document directly, you will need to change the "scope" field name in a JSON document to "scopes".
+
+= Liquibase upgraded to version 4.6.2
+
+Liquibase was updated from version 3.5.5 to 4.6.2, which includes, among other things, several bug fixes, and a new way of registering custom extensions
+via `ServiceLoader`.
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-17_0_0.adoc]: Change "via" (which is Latin) to "using" (which is English).

File: upgrading/topics/keycloak/changes-17_0_0.adoc [hunk 4]
```diff
@@ -22,6 +22,16 @@ A number of things have changed since the preview Quarkus distribution was relea
...
+
+Migration from previous Keycloak versions to Keycloak 17.0.0 has been extensively tested with all the currently supported databases,
```
Comment (andymunro) [inline upgrading/topics/keycloak/changes-17_0_0.adoc]: Remove "the" after "all"

**Review by andymunro — CHANGES_REQUESTED** [review]
Thanks, @sguilhen. Just a couple of minor changes.

**Comment by sguilhen** [comment]
@andymunro Thank you for the review. Just pushed a new version incorporating your suggestions.

**Review by andymunro — APPROVED** [review]
Thanks, @sguilhen

**Review by hmlnarik — APPROVED** [review]
Thanks for the PR!

**Pull request merged**